components/definitions/rewards.d.ts

```diff
@@ -208,4 +209,62 @@ declare namespace Rewards {
     rates: Record<string, number>
     wallets: Record<string, number>
   }
+
+  export interface AdsHistoryData {
+    [key: string]: any
```

Comment:
what does this one means?

Reply:
I'm not sure, but I think @jasonrsadler might know...